modules/claims_api/app/views/claims_api/_claims_status_table.html.erb

```diff
@@ -0,0 +1,26 @@
+<table>
+  <thead>
+    <tr>
+        <th>consumer</th>
+        <% ClaimsApi::AutoEstablishedClaim::ALL_STATUSES.each do |status| %>
+        <th><%= status %></th>
+        <% end %>
+        <th>totals</th>
+        <th>percentage with flashes</th>
+        <th>percentage with special issues</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% claims_consumers.each do |consumer| %>
+      <tr>
+        <td class="left-align"><%= consumer.keys[0] %></td>
+        <% ClaimsApi::AutoEstablishedClaim::ALL_STATUSES.map(&:to_sym).each do |status| %>
+        <td class="right-align"><%= consumer.values.first[status] || 0 %></td>
+        <% end %>
+        <td class="right-align"><%= consumer.values.first[:totals].to_i %></td>
+        <td class="right-align"><%= consumer.values.first[:percentage_with_flashes] %></td>
+        <td class="right-align"><%= consumer.values.first[:percentage_with_special_issues] %></td>
+      </tr>
+    <% end if claims_consumers %>
+  </tbody>
+</table>
```

Comment:
The reason this partial shows up as new is the formatting on the old one was wonky, so when I fixed it and moved it (along with the rest of the shared partials) it identified it as new.